### PR TITLE
Feature/ac 270940 2023 r1 bugfix t250 course update

### PR DESCRIPTION
**Pull request by MehranQadri**
Updates to code snippets and customization project code files based on current course update and previous code snippets update during conversion to chunks.

### CODE REVIEW

File: Customization/T250/PhoneRepairShop/Pages/RS/RS401000.aspx [hunk 1]
```diff
@@ -32,18 +32,18 @@
...
+              <px:PXGridColumn DataField="Status" />
+              <px:PXGridColumn DataField="InvoiceNbr" CommitChanges="True" />
+              <px:PXGridColumn DataField="PercentPaid" />
+              <px:PXGridColumn DataField="ARInvoice__DueDate" />
+              <px:PXGridColumn DataField="ARInvoice__CuryDocBal" />
           </Columns>
           <RowTemplate>
             <px:PXSelector ID="edInvoiceNbr" runat="server"
-                DataField="InvoiceNbr" Enabled="False" AllowEdit="True" />
+                DataField="InvoiceNbr" AllowEdit="True" />
```
Comment (k-s-popova) [inline Customization/T250/PhoneRepairShop/Pages/RS/RS401000.aspx:46]: Is that expected that Enabled="False" is removed?

File: Customization/T250/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVPaymentPlanInq.cs [hunk 2]
```diff
@@ -122,7 +122,7 @@ protected virtual void viewOrder()
                 var graph = PXGraph.CreateInstance<SOOrderEntry>();
                 // set the current property of the graph
                 graph.Document.Current = graph.Document.
-                  Search<RSSVWorkOrder.orderNbr>(order.OrderNbr);
+                  Search<SOOrder.orderNbr>(order.OrderNbr);
```
Comment (k-s-popova) [inline Customization/T250/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVPaymentPlanInq.cs:125]: Is that change expected? (just to check)

File: Customization/T250/PhoneRepairShop/Pages/RS/RS401000.aspx (hunk 1 above)
Comment (MehranQadri) [reply Customization/T250/PhoneRepairShop/Pages/RS/RS401000.aspx:46]: Yes. This change was made during T250 conversion to chunks based on SME feedback. See pull request #88

File: Customization/T250/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVPaymentPlanInq.cs (hunk 2 above)
Comment (MehranQadri) [reply Customization/T250/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVPaymentPlanInq.cs:125]: Same as above. Yes. This change was made during T250 conversion to chunks based on SME feedback.  See pull request #88

**Review by k-s-popova — APPROVED** [review]

**Pull request merged**